docker-compose.yml

```diff
@@ -115,27 +114,42 @@ services:
     restart: always
 
   grafana:
-    image: grafana/grafana:7.0.0
+    image: grafana/grafana:7.3.1
     environment:
       GF_SECURITY_ADMIN_PASSWORD: admin
       GF_USERS_ALLOW_SIGN_UP: 'false'
```

Comment:
This is only the initial launch password to get in. I've alread connected and changed it.